server/request_response.go

```diff
@@ -7,35 +7,29 @@ import (
 	"github.com/flashbots/rpc-endpoint/types"
 )
 
-func (r *RpcRequest) writeHeaderStatus(statusCode int) {
-	if r.respHeaderStatusCodeWritten {
-		return
-	}
-	r.respHeaderStatusCodeWritten = true
-	(*r.respw).WriteHeader(statusCode)
-}
+// As per JSON-RPC 2.0 Specification
+const (
+	ParseError     = -32700
```

Comment:
I would prefix those consts with `JsonRpc` or `JsonRpcError` to make it clearer what they are referring to.

Reply:
I have actually created `type jsonRPCError int` in the beginning later I removed the same. will do it

Reply:
updated